Gems/AWSCore/Code/Tools/ResourceMappingTool/resource_mapping_schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "type": "object",
+    "title": "The AWS Resource Mapping Root schema",
```

Comment:
"Root schema" seems strange. Maybe the "O3DE AWS Resource mapping file schema"?

We should be careful not to state "The AWS" anything, its not AWS's format, its the format for doing things with AWS resources.

Reply:
I see this a straight copy of https://docs.o3de.org/docs/user-guide/gems/reference/aws/aws-core/resource-mapping-schema/.  We should remove this from docs and just link to this file once its submitted.

Reply:
will update title, docs update should be a separate thing.